app/views/users/_sidebar.html.erb

```diff
@@ -5,7 +5,7 @@
   <li><%= span_if_current ts("Profile"), user_profile_path(@user) %></li>
   <% if @user.pseuds.size > 1 %>
     <li class="pseud" aria-haspopup="true">
-      <% pseud_link_text = current_page?(@user) ? ts("Pseuds") : (@pseud ? @pseud.name : @user.login) %>
+      <% pseud_link_text = current_page?(@user) ? ts("Pseuds") : (@pseud ? ( @pseud.name ? @pseud.name : ts("Pseuds") ) : @user.login) %>
       <a href="#" title="<%= ts("Pseud Switcher") %>"><%= pseud_link_text %></a>
```

Comment:
```suggestion
      <a href="#" title="<%= ts("Pseud Switcher") %>"><%= ts("Pseuds") %></a>
```

The good news is I was right and it is supposed to say Pseuds!

The bad news is, while I was checking the history to see what the logic _used_ to be to make sure I was understanding the ternary correctly, I discovered we actually clobbered #4127/[AO3-6249](https://otwarchive.atlassian.net/browse/AO3-6249) in the initial merge of this (#4554). We want it to say "Pseuds" _all_ the time, not just on the New Pseud and user page.

To make you slightly less inclined to want to murder me, I have submitted it as a suggestion so you can hopefully press the button and not need to do more work, but I'm happy to do penance in the form of opening up a PR if you'd rather.

Reply:
It sounds like the original-original intention of the code was to have the current pseud on pages that relate to that pseud, and the username was just a fallback for when the `@author` evaporated without us noticing.

I'm not sure the decision to have it say "Pseuds" at all times was reached with consideration for past designers' intentions and what would make the most sense for a user or visitor, or just as an "idk let's put something there, there are bigger fish and they're on fire" measure, but I'm reaching the point of not giving a flying fox anymore. (Which is bad and a sign of burnout, but whatever. The fish are bigger and so's the fire.)

Thanks for the code suggestion!